site/en/meet-the-team/events/events.11tydata.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  type: 'event',
+  permalink: false,
+  canLinkToList: true,
```

Comment:
Can you explaing what canLinkToList do? I see it is used in algolia, but I don't get the rationale behind it.

Reply:
As it stands currently, only items with a permalink are added to the index. We're not adding a permalink for events, but do want them to be included in the search results. The newly introduced variables, canLinkToList and listUrl allow this to happen.

Reply:
but what kind of List does it refer to? and what is listUrl?